### PR TITLE
af_unix: add function to remove duplicated codes for set sockaddr

### PR DESCRIPTION
**Pull request by 2xsec**
Hello,

A function for abstract unix socket is added to remove duplicated codes for setting sockaddr.

Thanks.

Signed-off-by: 2xsec <dh48.jeong@samsung.com>

### CODE REVIEW

**Comment by brauner** [comment]
jenkins: test this please

File: src/lxc/af_unix.c [hunk 1]
```diff
@@ -42,6 +42,34 @@
 
 lxc_log_define(af_unix, lxc);
 
+static size_t lxc_abstract_unix_set_sockaddr(struct sockaddr_un *addr,
```
Comment (brauner) [inline src/lxc/af_unix.c]: This needs to be `ssize_t` :)

Comment (2xsec) [reply src/lxc/af_unix.c]: @brauner I'll fix it. :)

**Comment by brauner** [comment]
jenkins: test this please

**Comment by brauner** [comment]
Thanks!

**Pull request merged**